app/main.ipynb

```diff
@@ -216,15 +271,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
-   "source": [
-    "# write your code here"
-   ],
+   "source": "np.std(A, axis=0)",
    "metadata": {
     "collapsed": false,
-    "is_executing": true
-   }
+    "is_executing": true,
+    "ExecuteTime": {
+     "end_time": "2024-11-07T10:08:42.159352Z",
+     "start_time": "2024-11-07T10:08:42.154042Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([0.89442719, 1.8547237 ])"
+      ]
+     },
```

Comment:
are you sure it's a correct answer? you have 5 values by Y axis but only 2 values are returned